src/client/cli/formatter/csv_formatter.cpp

```diff
@@ -92,7 +93,9 @@ std::string mp::CSVFormatter::format(const FindReply& reply) const
 
         mp::format::filter_aliases(aliases);
 
-        auto image_id = aliases[0].remote_name().empty() ? aliases[0].alias() : fmt::format("{}:{}", aliases[0].remote_name(), aliases[0].alias());
+        auto image_id = aliases[0].remote_name().empty()
+                            ? aliases[0].alias()
+                            : fmt::format("{}:{}", aliases[0].remote_name(), aliases[0].alias());
```

Comment:
Is this just a drive by `clang-format` change?

Reply:
That's what it looks like. Not seeing any other differences.

Reply:
Ok, thanks.